app/src/main/java/com/example/keniphhomemaintenance/dwellings/Dwelling.kt

```diff
@@ -1,6 +1,12 @@
 package com.example.keniphhomemaintenance.dwellings
 
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "dwellings_table")
 class Dwelling(
+    @PrimaryKey(autoGenerate = true)
```

Comment:
Not super familiar with autoGenerate - if we have that annotation, do we need the dwellingId field? If we do still need the id field, do we have to pass in a value for it when we create a new instance of Dwelling, since it's being autoGenerated?